bot/services/remind_service.py

```diff
@@ -0,0 +1,54 @@
+from bot.data.message_repository import MessageRepository
+from datetime import datetime
+from uuid import uuid4
+import logging
+
+import discord
+
+from bot.consts import Colors
+from bot.utils import converters
+from bot.messaging.events import Events
+from bot.services.base_service import BaseService
+from bot.data.remind_repository import RemindRepository
+
+log = logging.getLogger(__name__)
+
+
+class RemindService(BaseService):
+    def __init__(self, *, bot):
+        super().__init__(bot)
+        
+    @BaseService.Listener(Events.on_set_reminder)
+    async def on_set_reminder(self, userId: int, wait: converters.Duration, message: str, link: str):
+        id = str(uuid4())
```

Comment:
we dont need to generate a random ID sqlite will auto increment the pk for us

Reply:
Is there a way to retrieve this id once the item has been inserted as the id is used in the callback function?